### ISSUE TITLE
Scale bar in visualization

### ISSUE THREAD

**Issue by oanegros** [ilastik/ilastik#2870]
I would like a scale bar showing your current scale. 
This would require reading in pixel sizes, but that would be better anyway and you can assume some (i.e. imagej) standards.

**Comment by k-dominik** [comment]
related: https://github.com/ilastik/ilastik/issues/967

**Comment by oanegros** [comment]
Yes, this is related to integrating pixel size in all algorithms, but imo a lot easier to implement as you do not need to really deal with anisotropic kernels and backwards compatibility etc.
